user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Configuring_external_authentication_via_an_identity_provider_using_SAML.md

```diff
@@ -242,11 +242,12 @@ From DataMiner 10.2.0/10.1.12 onwards, users authenticated by Azure AD using SAM
      ...
    </DataMiner>
    ```
+1. User groups have to exist in DataMiner for users to be added to them (both for Groups claims set to true and false). See [Adding a user group] (Adding_a_user_group) for more info on how to do this
 
    > [!NOTE]
    > If you set the *claims* attribute of the *Groups* element to "false", no claims will be used to add users to groups. In this case:
    >
-   > - The name of the group as specified in Cube will be used instead.
+   > - Instead of a claim for user groups replace [group claim name] with a security group that exists in DataMiner as described above.
```

Comment:
Ideally, this note should be under the same step as the one containing that "[group claim name]", so that it's more clear what this is about.

Reply:
This refers specifically to what to do when not using a claim for group names so I think this is in the right spot
can't really put it next to the group claim name step as that would be more confusing and contradicting I think.